taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,495 @@
+{% extends "guide.html" %}
+
+{% block guide %}
+  <div class="row">
+    <div class="col">
+      <h2>A2CPS Data Transfer and Management Guide</h2>
```

Comment:
I hate to block this (and I know a2cps was the reference for this task), but since 3dem is the first up now, we may need to either: 
- Change hardcoded a2cps references to 3dem references (immediate, short term fix)
- Genericize this template for any portal (future, better)

Reply:
I chose "Genericize this template".

If I failed to achieve generic content, then I propose [FP-961](https://jira.tacc.utexas.edu/browse/FP-961) be assigned to appropriate knowledgeable people (perhaps @jgentle and @SilversunKSauri, who taught me enough to try for Globus and Rsync) and scheduled as necessary.